JSONSchema/schema.json

```diff
@@ -0,0 +1,168 @@
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "currentTimestamp":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "previousHash":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        }
+                    },
+                    "additionalproperties":false
+                },
+                "pre":{
+                    "type":"object",
+                    "patternProperties":{
+                        "^0x[0-9a-f]*":{
+                            "type":"object",
+                            "properties":{
+                                "balance":{
+                                    "type":"string",
+                                    "pattern":"^0x[0-9a-f]*$"
+                                },
+                                "code":{
+                                    "type":"string",
+                                    "pattern":"^(0x[0-9a-f]*)?$"
+                                },
+                                "nonce":{
+                                    "type":"string",
+                                    "pattern":"^0x[0-9a-f]*$"
+                                },
+                                "storage":{
+                                    "type":"object"
+                                }
+                            }
+                        }
+                    },
+                    "additionalProperties":false
+                },
+                "transaction":{
+                    "type":"object",
+                    "properties":{
+                        "data":{
+                            "type":"array",
+                            "items":{
+                                "type":"string",
+                                "pattern":"^(0x[0-9a-f]*)?$"
+                            }
+                        },
+                        "gasLimit":{
+                            "type":"array",
+                            "items":{
+                                "type":"string",
+                                "pattern":"^0x[0-9a-f]*$"
+                            }
+                        },
+                        "gasPrice":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "nonce":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "secretKey":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "to":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
```

Comment:
the `to` field can be empty, for a contract creation tx. [example](https://github.com/ethereum/tests/blob/3b90fbfaf0b6180be9276bbd050ac2475b95d949/GeneralStateTests/stInitCodeTest/StackUnderFlowContractCreation.json#L82)